src/content/docs/tutorial-app-performance/guide-to-monitoring-core-web-vitals.mdx

```diff
@@ -110,33 +133,9 @@ Each web vital has the following thresholds to help you track your app's perform
 
 **Good** web vitals can help your organization save money and increase SEO. **Poor** or **Needs improvement** can reflect negative user perceptions of your organization, which could result in dissatisfied customers or lost business opportunities.
 
-Want to learn more about how core web vitals are collected? Check out our
-[self-paced core web vitals training course](https://learn.newrelic.com/self-paced-core-web-vitals/). It's free!
-
-## Monitor core web vitals with New Relic [#monitor]
-
-Now that you're familiar with core web vitals, it's time to monitor them with New Relic.
-
-If you've already monitoring your browser app with New Relic, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Monitor > Web vitals**. 
+## Drill down into specific pages [#drill-down]
 
-If you're new to New Relic, [create a free New Relic account](https://newrelic.com/signup) and [install our browser agent](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent) (it takes around 5 minutes!). Then come back here to learn how to navigate the **Web vitals** page.
-
-<img
-  title="Track core web vitals site-wide, favorite key webpages, and filter by time period."
-  alt="overview of the web vitals page"
-  src={browserWebVitalsGif}
-/>
-<figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Monitor > Web vitals**: Track core web vitals site-wide, favorite key webpages, and filter by time period.
-</figcaption>
-
-<Callout variant="important">
-For some customers, **Web vitals** may be located under **More views** (instead of under **Monitor**).
-  </Callout>
-
-## Drill down into specific pages
-
-If you want to get insights into an important page, like an ecommerce checkout page, click on a page in the **Page URL** table. You'll see a breakdown of what's contributing to core web vitals for that specific page. Use this data to understand how your end users are experiencing your most important pages and what’s contributing to poor scores. 
+If you want insight into an important page — for example, an ecommerce checkout page — find it in the **Page URL** table. You'll see a breakdown of what's contributing to core web vitals, which you can use to understand how your end users are experiencing those areas of your site. For example, if you notice a pattern of poor performance from Chrome browsers, you can curate any future improvements to that user base. 
```

Comment:
Is it ecommerce or e-commerce? I actually don't know.

Reply:
! it's ecommerce: /docs/tutorial-app-performance/fixing-high-latency